src/Microsoft.TestPlatform.Common/Interfaces/Engine/ClientProtocol/TestExecutionContext.cs

```diff
@@ -44,7 +45,8 @@ public TestExecutionContext()
             bool areTestCaseLevelEventsRequired,
             bool hasTestRun,
             bool isDebug,
-            string testCaseFilter)
+            string testCaseFilter,
+            FilterOptions filterOptions)
```

Comment:
I'm hoping this would not break older versions of testhost. I do not think so. @navin22 ?

Reply:
I don't think so. 
@genlu Is these scenarios tested ? And also the Object Model changes in VS IDE TestExplorer scenarios need to be verified.